test/functional/JIT_Test/playlist.xml

```diff
@@ -717,6 +717,34 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>jit_recognizedCRC32C</testCaseName>
+		<variations>
+			<variation>-Xjit:count=1,disableAsyncCompilation,optlevel=scorching</variation>
```

Comment:
This would queue up every method that is involved in JVM init. to test framework for scorching level compilation. We certainly do not want. Is it possible to limit this just the test methods?